cmd/pkappa2/main.go

```diff
@@ -809,6 +809,35 @@ func main() {
 		}
 	})
 	rUser.Get("/*", http.FileServer(http.FS(&web.FS{})).ServeHTTP)
+	rUser.Get("/api/pcap_processors", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
we can make this generic, e.g.
/api/webhooks
in the future, we could add a type (which would be pcap_processed by default to support this version)